go/tools/gazelle/merger/fix.go

```diff
@@ -325,16 +378,30 @@ func FixLoads(oldFile *bf.File) *bf.File {
 	return &fixedFile
 }
 
-// knownKinds is the set of symbols that Gazelle ever generated loads for,
-// including symbols it no longer uses (e.g., cgo_library), not including
-// other symbols loaded manually (e.g., go_embed_data). This function will
-// only add or remove loads of these symbols.
-var knownKinds = map[string]bool{
-	"cgo_library": true,
-	"go_binary":   true,
-	"go_library":  true,
-	"go_prefix":   true,
-	"go_test":     true,
+// knownFiles is the set of labels for files that Gazelle loads symbols from.
+var knownFiles []string
+
+// knownKinds is a map from symbols to labels of the files they are loaded
+// from. All symbols Gazelle ever generated loads for are present, including
+// symbols it no longer uses (e.g., cgo_library). Manually loaded symbols
+// (e.g., go_embed_data) are not included.
+var knownKinds map[string]string
+
+func init() {
+	knownKinds = make(map[string]string)
+	for _, f := range []struct {
+		file  string
+		kinds []string
+	}{
+		{"go:def.bzl", []string{"cgo_library", "go_binary", "go_library", "go_prefix", "go_test"}},
+		{"proto:def.bzl", []string{"go_grpc_library", "go_proto_library"}},
+	} {
```

Comment:
I would declare this thing as a top level var (knownLoads?) near the two that are initialized from it, easy to find read and extend then.

Reply:
Done